utils/src/main/kotlin/CopyrightStatementsProcessor.kt

```diff
@@ -108,7 +108,7 @@ class CopyrightStatementsProcessor {
         val processedStatements: LinkedHashMap<String, List<String>>,
         val unprocessedStatements: List<String>
     ) {
-        fun toMutableSet() = (unprocessedStatements + processedStatements.keys).toMutableSet()
+        fun getStatements() = (unprocessedStatements + processedStatements.keys).toMutableSet()
```

Comment:
Nit: Maybe better "getAllStatements"? And BTW, does this really have to be mutable, as that's quite unusual for a temporary set?

Reply:
Thanks! Addressed both as suggested.